numpy/linalg/linalg.py

```diff
@@ -2112,6 +2112,9 @@ def norm(x, ord=None, axis=None, keepdims=False):
     """
     x = asarray(x)
 
+    if not issubclass(x.dtype.type, floating):
```

Comment:
Nope meant `floating`, but it wasn't imported. Just fixed as you were typing. :) See above.